lib/log.js

```diff
@@ -0,0 +1,94 @@
+var path = require('path')
+var fs = require('fs')
+var Console = require('console')
+var mkdir = require('mkdirp')
+var wsock = require('websocket-stream')
+var http = require('http')
+var through = require('through2')
+
+var nodeLogger = null
+var browserLogger = null
+
+var PORT = 17509
```

Comment:
Ok, leave for now. Let's leave a TODO. It's the same for the main process OSM API server anyway.